ui/store.js

```diff
@@ -96,7 +96,7 @@ const persistOptions = {
 
 let history;
 // @if TARGET='app'
-history = createHashHistory();
+history = createMemoryHistory();
```

Comment:
What is the difference between these two?

Reply:
createMemoryHistory gives us an array of entries so that we can reference the history stack.

*Excerpt from [history/docs/GettingStarted](https://github.com/ReactTraining/history/blob/master/docs/GettingStarted.md)*
> Additionally, createMemoryHistory provides history.index and history.entries properties that let you inspect the history stack.